### PR TITLE
Add ND CCTools

### PR DESCRIPTION
**Pull request by btovar**
The Cooperative Computing Tools (cctools) are a collection of programs
that enable large scale distributed computing on systems such as
clusters, clouds, and grids. These tools are commonly used in fields of
science and engineering that rely on large amounts of computing.

http://ccl.cse.nd.edu

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by btovar** [comment]
@conda-forge/help-c-cpp  This pr is ready to be review, thanks!

**Comment by btovar** [comment]
@conda-forge/staged-recipes This pr is ready to be reviewed, thanks!

File: recipes/ndcctools/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
```
Comment (marcelotrevisani) [inline recipes/ndcctools/build.sh]: Could you please remove this sheband?
``conda-build`` already manage it

File: recipes/ndcctools/build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+DISABLED=$(echo --without-system-{allpairs,parrot,prune,sand,umbrella,wavefront,weaver})
+
+if [[ $PY3K == 1 ]]; then
+    ./configure --prefix "${PREFIX}" --with-base-dir "${PREFIX}" --with-python3-path "${PREFIX}" --with-readline-path no  ${DISABLED}
+else
+    ./configure --prefix "${PREFIX}" --with-base-dir "${PREFIX}" --with-python-path "${PREFIX}" --with-readline-path no  ${DISABLED}
+fi
+
+make
```
Comment (marcelotrevisani) [inline recipes/ndcctools/build.sh]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/ndcctools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  rpaths:
+    - lib/
+
+requirements:
+  build:
+    # Compilers are named 'c', 'cxx', etc.
```
Comment (marcelotrevisani) [inline recipes/ndcctools/meta.yaml]: Could you please remove those comments?
```suggestion
```

File: recipes/ndcctools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  commands:
+    - makeflow -h
+    - work_queue_worker -h
+    - work_queue_factory -h
+
+about:
+  home: http://ccl.cse.nd.edu/
+  license: GNU GPL version 2
```
Comment (marcelotrevisani) [inline recipes/ndcctools/meta.yaml]:
```suggestion
  license: GPL-2.0
```

**Comment by btovar** [comment]
@marcelotrevisani, thanks for reviewing the pr!  I believe all that all the changes requested are completed.

**Comment by btovar** [comment]
@marcelotrevisani: force pushed to fix problem with updated feedstock.

**Pull request merged**